wallets/index.js

```diff
@@ -197,6 +187,13 @@ function useConfig (wallet) {
       }
 
       if (valid) {
+        try {
+          await wallet.testSendPayment?.(newConfig, { me, logger })
```

Comment:
For LNC, `testSendPayment` returns diffie hellman keys that need to be saved. You deleted this logic on line 80.

Reply:
I went ahead and fixed it.